server/src/main/java/org/apache/uniffle/server/ShuffleFlushManager.java

```diff
@@ -118,14 +123,24 @@ private void startEventProcessor() {
     processEventThread.start();
   }
 
-  protected Executor createFlushEventExecutor() {
+  protected void initFlushEventExecutor() {
```

Comment:
We shouldn't expose the concept of storageType in the FlushManager.

Reply:
Ok.